src/components/Sidebar.js

```diff
@@ -14,10 +14,10 @@ import { getSponsors } from '../utility/firebase'
 const SidebarContainer = styled.div`
   min-width: 275px;
   min-height: 100%;
-  border-right: 1px solid ${p => p.theme.colors.border};
+  // border-right: 1px solid ${p => p.theme.colors.border};
   transition: opacity 1s ease-out;
   z-index: 1;
-  background: ${p => p.theme.colors.background};
```

Comment:
Can you keep this line but comment it out so we know what it was like before? If we could do this for the other replaced CSS values that would be fantastic